src/utils/__tests__/get-code-from-ast.spec.ts

```diff
@@ -26,10 +24,7 @@ import a from 'a';
         directives: [],
     });
     expect(format(formatted, { parser: 'babel' })).toEqual(
-        `// first comment
-// second comment
-
-import a from "a";
+        `import a from "a";
```

Comment:
I'm guessing you removed the comments here because they're tested elsewhere and not directly related to this function, right?

Reply:
Exactly!